lib/siwapp/invoices/item.ex

```diff
@@ -14,8 +14,17 @@ defmodule Siwapp.Invoices.Item do
     field :description, :string
     field :unitary_cost, :integer, default: 0
     field :deleted_at, :utc_datetime
+    field :base_amount, :integer, virtual: true
+    field :discount_amount, :integer, virtual: true
+    field :net_amount, :integer, virtual: true
+    field :taxes_amount, :integer, virtual: true
+    field :gross_amount, :integer, virtual: true
```

Comment:
we'll talk about this. Too many fields for nothing

Reply:
I can remove the base_amount and the discount_amount fields. I think we need the other three in order to print the net_amount (base_amount - discount_amount), the taxes_amount for each tax and the gross_amount, which is the total.